app/connector/activemq/pom.xml

```diff
@@ -197,6 +197,24 @@
           </delimiters>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-remote-resources-plugin</artifactId>
```

Comment:
Perhaps instead of using this plugin we can simply point to the file in the repository for `jsonSchema` below?

Reply:
Refactored commit to extend maven-support-plugin.